.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: ruby
+
 rvm:
+  - "2.3.0"
```

Comment:
Travis doesn't have a `2.3` binary yet, so we need to specify the full version here.